packages/financial-templates-lib/src/clients/InsuredBridgeL2Client.ts

```diff
@@ -2,7 +2,7 @@
 // from the respective chains and return it to client implementors.
 
 import { getAbi } from "@uma/contracts-node";
-import type { OVMBridgeDepositBoxWeb3 } from "@uma/contracts-node";
+import type { BridgeDepositBoxWeb3 } from "@uma/contracts-node";
```

Comment:
unrelated change - made for consistancy. The client does not need `OVM` box, just the normal `BridgeDepositBox`